### PR TITLE
Add info for KDNet interface

### PR DESCRIPTION
**Pull request by abcSup**
## Detailed description

* Add instructions on how to set up and connect to a KDNet interface
* Reformat the guide

## Related Issues

radareorg/radare2#17340

### CODE REVIEW

**Review by wargio — APPROVED** [review]

File: src/debugger/windbg.md [hunk 1]
```diff
@@ -1,26 +1,22 @@
...
-Enable WinDBG support on Windows Vista and higher like this:
+## Setting Up KD on Windows
+
+### Serial Port
+Enable KD over a serial port on Windows Vista and higher like this:
 ```
 bcdedit /debug on
 bcdedit /dbgsettings serial debugport:1 baudrate:115200
 ```
-Starting from Windows 8 there is no way to enforce debugging
```
Comment (XVilka) [inline src/debugger/windbg.md:18]: Why did you remove this part?

Comment (abcSup) [reply src/debugger/windbg.md:18]: It is moved to the bottom.
